### ISSUE TITLE
Feature request: removal of attributes by regex

### ISSUE THREAD

**Issue by certainlyakey** [whitespace-se/storybook-addon-html#41]
Thanks for a great (and greatly implemented) addon! 

I have a request that will be familiar to anybody working with Angular. Angular adds a lot of attributes of its own into generated markup, like this: 
![image](https://user-images.githubusercontent.com/1833029/107160651-316f6580-69a0-11eb-8ba6-fabd384f8923.png)

Would it be possible to add a feature that would filter out an array of markup attributes by regex? So that we could strip out any attribute that begins from `_ng` or `ng-` and thus have a clean markup, leaving the Storybook consumers with all the parts that are really important.

**Comment by jeanfredrik** [comment]
Sounds like a good idea. Depending on your reason for showing HTML to the user it would be nice to decide exactly what attributes to hide yourself, like you suggest. I think we should add a parameter that filters all attributes and that can be set to either a blacklist array of strings, a regex or a filter function (that could also receive the tag name and attribute value for more advanced filtering).

**Comment by certainlyakey** [comment]
yes, that’s exactly what i meant 🙂 The regex feature for attribute names would be something that is definitely needed, since at least Angular generates quite random names.
